<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>972</width>
    <height>656</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Flock and 3D Herd Simulator v1.0</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="s_mainWindowGridLayout">
    <item row="1" column="1">
     <widget class="QGroupBox" name="s_drawGB">
      <property name="title">
       <string>Predators Behavioural Parameters</string>
      </property>
      <widget class="QCheckBox" name="huntB">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>90</y>
         <width>211</width>
         <height>22</height>
        </rect>
       </property>
       <property name="text">
        <string>Hunt B</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="gather">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>150</y>
         <width>211</width>
         <height>22</height>
        </rect>
       </property>
       <property name="text">
        <string>Gather</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="chill">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>60</y>
         <width>201</width>
         <height>22</height>
        </rect>
       </property>
       <property name="text">
        <string>Chill!</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QCheckBox" name="protPos">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>180</y>
         <width>121</width>
         <height>22</height>
        </rect>
       </property>
       <property name="text">
        <string>Protect Position</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="huntA">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>120</y>
         <width>211</width>
         <height>22</height>
        </rect>
       </property>
       <property name="text">
        <string>Hunt A</string>
       </property>
      </widget>
      <widget class="QSpinBox" name="predatornum">
       <property name="geometry">
        <rect>
         <x>180</x>
         <y>40</y>
         <width>48</width>
         <height>25</height>
        </rect>
       </property>
       <property name="value">
        <number>20</number>
       </property>
      </widget>
      <widget class="QDoubleSpinBox" name="m_positionY">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>210</y>
         <width>61</width>
         <height>25</height>
        </rect>
       </property>
       <property name="minimum">
        <double>-360.000000000000000</double>
       </property>
       <property name="maximum">
        <double>360.000000000000000</double>
       </property>
       <property name="value">
        <double>300.000000000000000</double>
       </property>
      </widget>
      <widget class="QDoubleSpinBox" name="m_positionX">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>210</y>
         <width>61</width>
         <height>25</height>
        </rect>
       </property>
       <property name="minimum">
        <double>-360.000000000000000</double>
       </property>
       <property name="maximum">
        <double>360.000000000000000</double>
       </property>
       <property name="value">
        <double>300.000000000000000</double>
       </property>
      </widget>
      <widget class="QDoubleSpinBox" name="m_positionZ">
       <property name="geometry">
        <rect>
         <x>170</x>
         <y>210</y>
         <width>61</width>
         <height>25</height>
        </rect>
       </property>
       <property name="minimum">
        <double>-360.000000000000000</double>
       </property>
       <property name="maximum">
        <double>360.000000000000000</double>
       </property>
       <property name="value">
        <double>300.000000000000000</double>
       </property>
      </widget>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>250</y>
         <width>91</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Protect Radius</string>
       </property>
      </widget>
      <widget class="QDoubleSpinBox" name="protRadius">
       <property name="geometry">
        <rect>
         <x>160</x>
         <y>250</y>
         <width>68</width>
         <height>25</height>
        </rect>
       </property>
       <property name="maximum">
        <double>4000.000000000000000</double>
       </property>
       <property name="singleStep">
        <double>10.000000000000000</double>
       </property>
       <property name="value">
        <double>200.000000000000000</double>
       </property>
      </widget>
      <widget class="QLabel" name="label_6">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>40</y>
         <width>131</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Number of Predators:</string>
       </property>
      </widget>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QGroupBox" name="s_transformGB">
      <property name="title">
       <string>Flock Behavioural Parameters</string>
      </property>
      <widget class="QSlider" name="separation">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>100</y>
         <width>84</width>
         <height>16</height>
        </rect>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
      <widget class="QCheckBox" name="evadeS">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>180</y>
         <width>122</width>
         <height>22</height>
        </rect>
       </property>
       <property name="text">
        <string>Evade as Sheep</string>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
      </widget>
      <widget class="QSlider" name="alignment">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>130</y>
         <width>84</width>
         <height>16</height>
        </rect>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
      <widget class="QSpinBox" name="boidnum">
       <property name="geometry">
        <rect>
         <x>177</x>
         <y>31</y>
         <width>48</width>
         <height>25</height>
        </rect>
       </property>
       <property name="maximum">
        <number>500</number>
       </property>
       <property name="value">
        <number>50</number>
       </property>
      </widget>
      <widget class="QCheckBox" name="evadeP">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>160</y>
         <width>111</width>
         <height>22</height>
        </rect>
       </property>
       <property name="text">
        <string>Evade as Prey</string>
       </property>
      </widget>
      <widget class="QSlider" name="cohesion">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>70</y>
         <width>84</width>
         <height>16</height>
        </rect>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>70</y>
         <width>71</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Cohesion</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>100</y>
         <width>71</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Separation</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_4">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>130</y>
         <width>71</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Alignment</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_5">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>40</y>
         <width>111</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Number of Boids:</string>
       </property>
      </widget>
      <widget class="QPushButton" name="reset">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>250</y>
         <width>80</width>
         <height>24</height>
        </rect>
       </property>
       <property name="text">
        <string>Reset</string>
       </property>
      </widget>
      <widget class="QPushButton" name="exportData">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>250</y>
         <width>111</width>
         <height>24</height>
        </rect>
       </property>
       <property name="text">
        <string>Export Data</string>
       </property>
      </widget>
     </widget>
    </item>
    <item row="0" column="0">
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>972</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
